.freeCodeCamp/config/projects.json

```diff
@@ -4,7 +4,7 @@
     "title": "Learn Digital Ledgers by Building a Blockchain",
     "dashedName": "learn-digital-ledgers-by-building-a-blockchain",
     "isIntegrated": false,
-    "description": "Placeholder description of project until Tom can think of a clever one.",
+    "description": "In this course, you will learn how to build a blockchain from scratch using JavaScript.",
```

Comment:
Slightest of suggestions: Perhaps we remove all the `In this course`s?

If it sounds too odd to just have _"You will learn..."_, we could go with _"Learn how to..."_, or _"This course teaches you how to..."_

Reply:
We could potentially remove all those - I don't like them all that much either, too repetitive. But it does let users know if it's a course or project, and it's also pretty much how the descriptions on /learn start - so I could go either way.